include/nds/arm9/rumble.h

```diff
@@ -32,14 +32,21 @@
 extern "C" {
 #endif
 
-#define RUMBLE_PAK			(*(vuint16 *)0x08000000)
+#define RUMBLE_PAK		(*(vuint16 *)0x08000000)
 #define WARIOWARE_PAK		(*(vuint16 *)0x080000C4)
 #define WARIOWARE_ENABLE	(*(vuint16 *)0x080000C6)
 
 typedef enum {
-   RUMBLE,
-   WARIOWARE
-}RUMBLE_TYPE;
+   RUMBLE_TYPE_UNKNOWN,
+   RUMBLE_TYPE_NONE,
+   RUMBLE_TYPE_PAK, /* DS Rumble Pak */
+   RUMBLE_TYPE_GBA, /* rumble included as part of GBA game cartridges */
+   RUMBLE_TYPE_MAGUKIDDO /* rumble/sensor cartridge bundled with Magukiddo */
+} RUMBLE_TYPE;
```

Comment:
Breaking these definitions is probably fine, as they were never exposed to user code before anyway.